test/domain/user/userControllerTest.js

```diff
@@ -0,0 +1,194 @@
+require('rootpath')();
+
+const sinon          = require('sinon');
+const chai           = require('chai');
+const expect         = chai.expect;
+const chaiAsPromised = require("chai-as-promised");
+
+chai.use(chaiAsPromised);
+
+const HttpStatus = require('http-status-codes');
+
+const NotFoundError  = require('error/notFoundError');
+const UserService    = require('domain/user/userService');
+const UserController = require('domain/user/userController');
+
+describe('UserController', function() {
+
+    let sandbox;
+
+    let userController;
+
+    before(async function() {
+        const userService = new UserService();
+        userController    = new UserController(userService);
+    });
+
+    beforeEach(function() {
+        sandbox = sinon.createSandbox();
```

Comment:
If you're using latest version of sinon, it has 'default' sandbox. You don't have to create it manually, just create your usual stubs and then use 'sinon.restore'.